master/reference/advanced/etcd-rbac/kubernetes.md

```diff
@@ -0,0 +1,85 @@
+---
+title: etcd RBAC with Kubernetes and Calico
```

Comment:
How about something like:  `Using etcd RBAC to segment Kubernetes and Calico data`

I'd find that a little more descriptive.  or another word instead of segment.